hveto/utils.py

```diff
@@ -19,10 +19,17 @@
 """General utilities for hveto
 """
 
+import glob
+import warnings
+
+from hveto import const
+
 from math import ceil
 
 from gwdatafind.utils import filename_metadata
 
+from gwpy.table import EventTable, Column
+
```

Comment:
This is probably splitting hairs, but for general consistency, can you re-order these imports slightly?

```python
import glob
import warnings

from math import ceil
from gwdatafind.utils import filename_metadata

from gwpy.table import (Column, EventTable)

from . import const
```

A few things to note:
* This order goes: native python modules first, then third-party modules like numpy and scipy, then submodule imports, then software-specific applications in hierarchical order: gwdatafind, then gwpy, then this current package (hveto)
* In general it's good practice to explicitly wrap imported modules in a tuple, order them from classes to functions/modules, and alphabetize (e.g. `from gwpy.table import (Column, EventTable)` rather than `from gwpy.table import EventTable, Column`)
* The change to `from . import const`, rather than `from hveto import const`, is actually a functional change. The former means "import this module directly from the current folder," the latter means "import this module from its install path." In most circumstances there should be no difference, but if you're testing changes, etc. this is a very profound difference.

Reply:
This is very useful. Thanks for the explanation.